backend/src/interface/mealplan_management/mod.rs

```diff
@@ -0,0 +1,11 @@
+//! This interface allows starting the operations for updating the menu from the the canteen's website.
+
+pub trait MensaParseScheduling {
```

Comment:
I think these methods should also be async.

Reply:
No, the functions have no returns

Reply:
In rust, a function without explicit return type returns `()`. You cannot await a async function in a synchronous one.